README.md

```diff
@@ -201,3 +200,22 @@ request.AddHeader("Content-Type", "application/json");
 request.AddParameter("undefined", "{\n\t\"to\" : \"+250781234567\", \n\t\"text\" : \"Hello from Pindo\",\n\t\"sender\" : \"Pindo\"\n}", ParameterType.RequestBody);
 IRestResponse response = client.Execute(request);
 ```
+
+```ruby
+# ruby
+
+require 'net/http'
+require 'json'
+
+uri = URI('http://api.pindo.io/v1/sms/')
+data = { to: '+250781234567', text: 'Hello from Pindo', sender: 'Pindo' };
+# To send the request via https, keep the use_ssl to true
+res = Net::HTTP.start(uri.host, uri.port, use_ssl: true) do |http|
```

Comment:
@OlivierJM our API doesn't support SSL for now, but a stable release of the product is scheduled for this month.
I suggest we remove SSL for the meantime.
```ruby
http = Net::HTTP.new(url.host, url.port);
request = Net::HTTP::Post.new(url)
```

Reply:
yeah that works, let me update that.